src/components/Tracking.tsx

```diff
@@ -31,11 +36,55 @@ export interface TrackingProps {
   onMouseUp?: (result: TrackingResult) => void;
   onMouseDown?: (result: TrackingResult) => void;
   onDoubleClick?: (result: TrackingResult) => void;
+  onWheel?: (result: TrackingResult) => void;
+  onKeyPress?: (result: KeysResult) => void;
+  onKeyDown?: (result: KeysResult) => void;
+  onKeyUp?: (result: KeysResult) => void;
   stateSeries: PlotSeriesState[];
 }
 
 const HORIZONTAL = ['bottom', 'top'];
+function infoFromKey(
+  event: React.KeyboardEvent<SVGRectElement>,
+  axisContext: Record<string, PlotAxisContext>,
+  plotHeight: number,
+  plotWidth: number,
+) {
+  const { scale: scaleY } = axisContext.y;
+  const { scale: scaleX } = axisContext.x;
+  return {
+    event,
+    x: { min: scaleX.invert(0), max: scaleX.invert(plotWidth) },
+    y: { min: scaleY.invert(plotHeight), max: scaleY.invert(0) },
+  };
+}
+function infoFromWheel(
+  event: React.WheelEvent<SVGRectElement>,
+  axisContext: Record<string, PlotAxisContext>,
+  plotHeight: number,
+) {
+  const ratio = 1 + event.deltaY * -0.001;
```

Comment:
Sorry my fault. The issue was not clear enough.
Actually the vertical scaling should, in this case, keep the 0 where it is.
So you should always change minY and maxY in order to keep the 0 at the same place and you may ignore the position of the mouse.

Reply:
I fixed the y of the mouse while zooming
So the user can zoom where he wants to